membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -212,17 +209,16 @@ object AccountDetails {
     }
     loop(start)
   }
-  def mmaCategoryFrom(product: Product): String = product match {
-    case _: Product.Paper => "subscriptions" // Paper includes GW 🤦‍
-    case Product.Digipack => "subscriptions"
-    case Product.SupporterPlus => "recurringSupport"
-    case Product.TierThree => "recurringSupport"
-    case Product.AdLite => "subscriptions"
-    case Product.GuardianPatron => "subscriptions"
-    case Product.Contribution => "recurringSupport"
-    case Product.Membership => "membership"
-    case _ => "subscriptions" // fallback - passing undefined value breaks manage
-  }
+
+  def getTier(catalog: Catalog, plan: RatePlan): Json.JsValueWrapper =
+    if (
+      plan.product(catalog) == Product.Delivery &&
```

Comment:
only covers Home delivery at the moment, as we don't think voucher or ND has that option (although we can check the data and update in another PR)